specify/specify7
# Highlight preference text in app resource

Fixes #4496

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

open app resource 
open global preferences 
verify that the text color is present
